utils/textnorm_zh.py

```diff
@@ -773,6 +804,8 @@ def nsw_test():
         if args.remove_fillers:
             for ch in FILLER_CHARS:
                 text = text.replace(ch, '')
+                
+            text = remove_er(text,ALLOW_ER_WRD)    
```

Comment:
can you add a flag to control Er Hua elimination? such as:
```
if args.remove_erhua:
    ...
```
and ErHua removal should be of the same level as remove_fillers, not inside.

Reply:
I have changed it